beta-src/src/utils/map/getOrdersMeta.ts

```diff
@@ -62,6 +75,20 @@ export default function getOrdersMeta(data, phase): Props {
             }
             return false;
           });
+          if (phase === "Retreats") {
+            const occupiedTerritory = newBoard.territories.find(
+              (t) => t.id === orderUnit.terrID,
+            );
+            allowedBorderCrossings = allowedBorderCrossings.filter(
+              (crossing) => !crossing.unitID && !crossing.standoff,
+            );
+            if (occupiedTerritory) {
+              allowedBorderCrossings = allowedBorderCrossings.filter(
+                (crossing) =>
+                  crossing.id !== occupiedTerritory.occupiedFromTerrID,
+              );
+            }
+          }
```

Comment:
it appears we are re-assigning allowedBorderCrossings too much, we should move the occupiedTerritory check into the filter above if possible.